### ISSUE TITLE
Add: ProTVChisinau.ro

### ISSUE THREAD

**Issue by ovidiuux** [iptv-org/iptv#11234]
### Channel ID

ProTVChisinau.ro

### Stream URL

http://hls.protv.md/hls/protv.m3u8

### Notes

24/7 from ProTV MD website.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Thank you for the stream: it will be included in the next release.

**Comment by LaneSh4d0w** [comment]
Nope, this one loops over and over to the same sequence. Hasn't changed since 2022

**Comment by freearhey** [comment]
http://hls.protv.md/hls/protv.m3u8
<img width="256" alt="image" src="https://user-images.githubusercontent.com/7253922/232255198-af134315-9c9a-47a2-8704-55056b677b1f.png">
